examples/Basics/recommenders.py

```diff
@@ -55,15 +52,8 @@
 # This model uses available data to model the objective function as well as the uncertainty.
 # The surrogate model is then used by the acquisition function to make recommendations.
 
-# The following are some available basic surrogates
-# Use `baybe.surrogates.get_available_surrogates()` for a complete list.
```

Comment:
@Scienfitz, @AVHopp. As you can see, I've remove the `get_available_surrogates` method, whose original purpose was only to serve as a workaround for `get_subclasses` not working due to the problems that have now been removed in this PR.

It's now debatable whether we want to bring that function (but now only as a convenience function) because you'll see that otherwise some additional filtering a la
```python
[cls() for cls in get_subclasses(Surrogate) if cls.__name__ != "CustomONNXSurrogate"]
```
needs to be done often because the ONNX model is somewhat special. On the other hand, I'm not sure if an end user would actually need/use that function. For instance, in situations like the example here, I would not even call it but rather point to the doc page directly, where all surrogates are listed and will also be explained in the future. This gives a much richer info anyway ...

Reply:
If you think that it might be useful as a convenience function for us developers, I see nothing against keeping it, but making it private. If we see that it is not really useful or helpful, we can still remove it later.